components/create_team/components/display_name.test.tsx

```diff
@@ -60,7 +60,7 @@ describe('/components/create_team/components/display_name', () => {
             },
         };
 
-        wrapper.find('.form-control').instance().value = teamDisplayName;
+        (wrapper.find('.form-control') as unknown as ReactWrapper<any, any, HTMLInputElement>).instance().value = teamDisplayName;
```

Comment:
Now I'm just making stuff up that might work
```suggestion
        wrapper.find<HTMLInputElement>('.form-control').instance().value = teamDisplayName;
```

Reply:
Nope lol